features/core/src/main/java/org/apache/karaf/features/internal/service/FeatureConfigInstaller.java

```diff
@@ -252,12 +252,16 @@ private void installConfigurationFile(String fileLocation, String finalname, boo
         finalname = substFinalName(finalname);
 
         File file = new File(finalname);
+        if (!file.getCanonicalPath().startsWith(System.getProperty("karaf.base"))) {
```

Comment:
Will that prevent me from installing a config under "karaf.etc" which is not under "karaf.base" ?

Reply:
Indeed, I think it's better to use `karaf.etc` rather than `karaf.base`.